string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,27 @@
+/* exported ransomCase */
+
+/*
+make variable with first letter lower cased | cased
+look at each letter in string after the first
+if index is even AND is uppercased | test case QUUX
+  concatenate lower case letter at index to cased
+if index is even
+  concatenate lower case letter at index to cased
+else
+  concatenate upper case letter at index to cased
+return cased
+*/
+
+function ransomCase(string) {
+  var cased = string[0].toLowerCase();
+  for (var i = 1; i < string.length; i++) {
+    if (i % 2 === 0 && string[i] === string[i].toUpperCase()) {
+      cased += string[i].toLowerCase();
+    } else if (i % 2 === 0) {
```

Comment:
What is the difference between these two blocks?

Reply:
there's a much simpler logic if I account for not even's first! I got stuck on this one and tried to code for specific test cases that didn't pass 😅